packages/jellyfish-api-core/__tests__/category/account/listCommunityBalances.test.ts

```diff
@@ -0,0 +1,30 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { BigNumber } from '@defichain/jellyfish-json'
+
+describe('Account', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should listCommunityBalances', async () => {
+    const data = await client.account.listCommunityBalances()
+
+    expect(data.AnchorReward instanceof BigNumber).toStrictEqual(true)
+    expect(data.IncentiveFunding instanceof BigNumber).toStrictEqual(true)
+    expect(data.Swap instanceof BigNumber).toStrictEqual(false)
+    expect(data.Futures instanceof BigNumber).toStrictEqual(false)
+    expect(data.Options instanceof BigNumber).toStrictEqual(false)
+    expect(data.Unallocated instanceof BigNumber).toStrictEqual(false)
+    expect(data.Unknown instanceof BigNumber).toStrictEqual(false)
+    expect(data.Burnt instanceof BigNumber).toStrictEqual(true)
```

Comment:
this is quite extreme for me, but its good, normally we just assert what's the data exactly return
if you want assert the undefined then
```
expect(data.AnchorReward instanceof BigNumber).toStrictEqual(true)
expect(data.IncentiveFunding instanceof BigNumber).toStrictEqual(true)
expect(data.Burnt instanceof BigNumber).toStrictEqual(true)

expect(data.Swap).toBeUndefined()
expect(data.Futures).toBeUndefined()
... blah blah blah
```